diff --git a/src/dbus/com.deepin.Manual.Open.service b/src/dbus/com.deepin.Manual.Open.service
index eef25b5f..b1abe387 100755
--- a/src/dbus/com.deepin.Manual.Open.service
+++ b/src/dbus/com.deepin.Manual.Open.service
@@ -1,3 +1,4 @@
 [D-BUS Service]
 Name=com.deepin.Manual.Open
-Exec=/usr/bin/dman --dbus
\ No newline at end of file
+#Exec=/usr/bin/dman --dbus
+/usr/bin/ll-cli run org.deepin.manual --exec "dman --dbus"
diff --git a/src/dbus/com.deepin.Manual.Search.service b/src/dbus/com.deepin.Manual.Search.service
index 1729b5b5..f1c29593 100755
--- a/src/dbus/com.deepin.Manual.Search.service
+++ b/src/dbus/com.deepin.Manual.Search.service
@@ -1,3 +1,4 @@
 [D-BUS Service]
 Name=com.deepin.Manual.Search
-Exec=/usr/bin/dmanHelper
\ No newline at end of file
+#Exec=/usr/bin/dmanHelper
+/usr/bin/ll-cli run org.deepin.manual --exec "dmanHelper"
