diff -Naur a/src/CMakeLists.txt b/src/CMakeLists.txt
--- a/src/CMakeLists.txt	2022-04-21 15:08:04.326104724 +0800
+++ b/src/CMakeLists.txt	2022-04-21 15:08:44.978102676 +0800
@@ -3,9 +3,10 @@
 #set(CMAKE_C_COMPILER  /usr/bin/clang)
 #set(CMAKE_CXX_COMPILER  /usr/bin/clang++)
 
-
+if(CMAKE_INSTALL_PREFIX)
+else()
 set(CMAKE_INSTALL_PREFIX /usr)
-
+endif()
 
 if (CMAKE_BUILD_TYPE MATCHES Debug)
   add_definitions(-DDMAN_SEARCH_DB="${CMAKE_SOURCE_DIR}/manual-assets")
